code/controllers/subsystems/ticker.dm

```diff
@@ -176,8 +176,8 @@ SUBSYSTEM_DEF(ticker)
 	Master.SetRunLevel(RUNLEVEL_GAME)
 	create_characters() //Create player characters and transfer them
 	collect_minds()
-	equip_characters()
 	move_characters_to_spawnpoints()
+	equip_characters()
```

Comment:
i don't understand the logic behind moving people to spawn after equipping them. Won't this result in people spawning as naked bald people for a second before their clothes appear?

Reply:
no, there shouldnt be any ticks between that and this is intended way to do it
otherwise loadout items are dropped in null space